packages/scene-composer/src/components/panels/scene-components/common/ValueDataBindingBuilder.tsx

```diff
@@ -91,8 +91,13 @@ export const ValueDataBindingBuilder: React.FC<IValueDataBindingBuilderProps> =
     // Initiate the provider
     const state = valueDataBindingStore.setBinding(componentRef, binding, dataBindingConfig);
     setBuilderState(filterBuilderState(state));
-    setAutoSuggestValue(state.selectedOptions[ENTITY_ID_INDEX]?.value || autoSuggestValue);
-  }, [componentRef, binding, valueDataBindingProvider, dataBindingConfig]);
+  }, [binding, valueDataBindingProvider, dataBindingConfig]);
+
+  useEffect(() => {
+    const state = valueDataBindingStore.setBinding(componentRef, binding, dataBindingConfig);
+    setBuilderState(filterBuilderState(state));
+    setAutoSuggestValue(state.selectedOptions[ENTITY_ID_INDEX]?.value || '');
+  }, [componentRef]);
```

Comment:
So the state is still dependent on binding and databindingConfig in this new useEffect but their not in the dependency array.  What's the advantage of splitting this into two use effects? Could you get the same effect just by updating the setAutoSuggestValue parameter to (value || '') instead?  What's the benefit of changing from autoSuggestValue last value vs ''?